.rubocop.yml

```diff
@@ -18,6 +18,12 @@ AllCops:
     - 'truffleruby/target/classes/**/*'
     - 'mxbuild/**/*'
 
+Lint/AssignmentInCondition:
+  Description: Don't use assignment in conditions.
+  StyleGuide: "#safe-assignment-in-condition"
+  Enabled: true
+  AllowSafeAssignment: true
+
```

Comment:
No I don't need `(=)`, nor I particularly like it, still I recognise that it's more distinguishing. We are also not the only ones reading the code. It improves the code, since it's reduces a chance to overlook assignment. It improves consistency since it's a more strict rule.